CONTRIBUTING.md

```diff
@@ -30,14 +30,14 @@ To get started, please [sign](https://cla-assistant.io/tldr-pages/tldr) the
 
 ## Guidelines
 
-The basic format of a `tldr` page is a set of concrete usage examples.
+The basic format of a `TLDR` page is a set of concrete usage examples.
 Here are a few guidelines to get started:
 
 1. Try to keep pages at around 5 examples. Pages can be longer if needed, but don't exceed 8 examples.
    Remember, it's OK if the page doesn't cover everything; that's what `man` is for.
 2. When in doubt, keep new command-line users in mind. Err on the side of clarity rather than terseness.
    For example, commands that require `sudo` should include it directly in the examples.
-3. Try to incorporate the spelled-out version of single-letter options in the example's description.
+3. Try to incorporate the spelt-out version of single-letter options in the example's description.
```

Comment:
Nice catch.

Reply:
Except this change, let's revert all the others.

Reply:
It's not actually incorrect, it's just [one of the valid variants](https://en.wiktionary.org/wiki/spelled) for the past participle of "spell".